travis/coverage.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -ev
+if [ $TRAVIS_BRANCH != "cesium.com" ]; then
```

Comment:
A potentially cleaner way of accomplishing this could be to use this condition to omit the stage entirely, like so:

```
    - stage:
      name: "Coverage"
      node_js: "18"
      if: repo != CesiumGS/cesium OR branch != cesium.com
      script:
        - ./travis/coverage.sh
```

However, I'm not sure if this omission of the stage would be confusing. We already have a lot of branching in the CI scripts and it might not be the best ideas to introduce the conditions in two different ways. I'm looking for feedback for this.

Reply:
I _think_ we still want to run the coverage step itself in non "CesiumGS/cesium" repos, as it's the only non-release version of the test suite that's run. I can see an argument for how that may not be catching anything but niche test failures, but I think it would be useful to provide an external contributor the test failure sooner in the process to avoid extra cycles of review.

Reply:
Ok, that sounds good to me.